README.md

```diff
@@ -247,6 +247,12 @@ $ hack/build-images.sh
 
 To hack on the web console, check out the [assets/README.md](assets/README.md) file for instructions on testing the console and building your changes.
 
+Security Response
+-----------------
+If you've found a security issue that you'd like to disclose confidentially
+please contact secalert@redhat.com and someone from Red Hat's security response
+team will get back to you as soon as possible.
```

Comment:
link to https://access.redhat.com/security/team/contact ? @kseifriedredhat is that the best destination to link to?